### ISSUE TITLE
Need clear (and automated?) GitHub onboarding for new members

### ISSUE THREAD

**Issue by astearns** [w3c/wg-effectiveness#72]
Getting a new working group member access to our repos is an undocumented, manual, frustrating process. We should fix the process, then document it here:

https://w3c.github.io/faq.html

I currently have a "new" working group member who needs write access to our repos (he actually joined 4 months ago, but he's just now taking on some editing responsibilities). He knew that he needed to link his W3C account to his GitHub account, so that's all good.
I can't add him to our CSSWG team, because his GitHub account isn't part of the W3C organization. I can't add him to the org, because I don't have the right permissions. So I need to send an email to our team contacts to get this done, and ask our new editor to wait.

I think there should be an automated process that gets triggered when someone joins a working group and/or links their W3C account with a GitHub account. If they've joined and already linked (or link and are already a member) then the GitHub account should automatically get added to the org.

If we can't automate this, then it should be a checklist item that gets done ASAP once someone joins a group. Waiting on the manual process until the need for editing comes up makes for a frustrating first-edit experience.

**Comment by RachelComerford** [comment]
Agreed - I was very new to GitHub when I came to the W3C and found the whole process very confusing. I was lucky enough to have a colleague who tutored me in merges, pull requests, and the like but without that I would have felt a little desperate.

**Comment by nrooney** [comment]
plh stated this is being worked on
